### PR TITLE
Create ru-board.com

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

**Comment by pipboy96** [comment]
Please also try other ru-board.com domains.

**Comment by TotalCaesar659** [comment]
How can I get the list of other domains? It redirects me to forum.ru-board.com.

**Comment by pipboy96** [comment]
Use Amass: https://github.com/OWASP/Amass

**Comment by pipboy96** [comment]
This is a list I got by running Amass:

```
gallery.ru-board.com
i.ru-board.com
mail.ru-board.com
host1.ru-board.com
newhost.ru-board.com
forall.ru-board.com
smilies.ru-board.com
shop.ru-board.com
chess.ru-board.com
a.ru-board.com
ru-board.com
tracker.ru-board.com
dc.ru-board.com
www.forum.ru-board.com
forum.ru-board.com
mail2.ru-board.com
front2.ru-board.com
i2.ru-board.com
mail3.ru-board.com
ib.ru-board.com
front1.ru-board.com
rss.ru-board.com
gazeta.ru-board.com
gal.ru-board.com
test.ru-board.com
mygui.ru-board.com
host3.ru-board.com
www.ru-board.com
mods.ru-board.com
ns1.ru-board.com
ns2.ru-board.com
www.dc.ru-board.com
forum2.ru-board.com
```

**Comment by TotalCaesar659** [comment]
Thank you very much!

File: src/chrome/content/rules/ru-board.com.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		test.ru-board.com
+
+	Time out:
+		dc.ru-board.com
+		www.dc.ru-board.com
+		mail3.ru-board.com
+		tracker.ru-board.com
+-->
+<ruleset name="ru-board.com">
+	<target host="ru-board.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/ru-board.com.xml]: expired？

File: src/chrome/content/rules/ru-board.com.xml [hunk 2]
```diff
@@ -33,7 +34,6 @@
 		tracker.ru-board.com
 -->
 <ruleset name="ru-board.com">
-	<target host="ru-board.com" />
 	<target host="www.ru-board.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/ru-board.com.xml]: expired either

**Pull request merged**